src/aoc2020/day3_update.clj

```diff
@@ -0,0 +1,103 @@
+(ns aoc2020.day3-update
+  (:require [aoc2020.util :refer [file->seq]]))
+
+(defn grid-template->grid
+  [grid-template]
+  (mapv (fn [row] (cycle (seq row)))
+        grid-template))
+
+(defn position-s
+  [{slope-right :right
+    slope-down  :down}]
```

Comment:
You'll generally want to `{:keys ..` form. 
As you are controlling the map coming in, you should be asking why the names of the keys are mis aligned with meaning